gooddata-sdk/gooddata_sdk/catalog/organization/service.py

```diff
@@ -510,6 +516,162 @@ def list_export_templates(self) -> list[CatalogExportTemplate]:
             for export_template in export_templates["data"]
         ]
 
+    def get_llm_endpoint(self, id: str) -> CatalogLlmEndpoint:
+        """
+        Get LLM endpoint by ID.
+
+        Args:
+            id: LLM endpoint identifier
+
+        Returns:
+            CatalogLlmEndpoint: Retrieved LLM endpoint
+        """
+        response = self._entities_api.get_entity_llm_endpoints(id, _check_return_type=False)
+        return CatalogLlmEndpoint.from_api(response.data)
+
+    def list_llm_endpoints(
+        self,
+        filter: str | None = None,
+        page: int | None = None,
+        size: int | None = None,
+        sort: list[str] | None = None,
+        meta_include: list[str] | None = None,
```

Comment:
Please, replace `something | None` with `Optional[something]`. Python SDK still has support for Python 3.9, and [PEP-604](https://peps.python.org/pep-0604/) was introduced in Python 3.10